### ISSUE TITLE
Convergence of gene-names for annotation and downstream filter

### ISSUE THREAD

**Issue by mathiasbio** [Clinical-Genomics/BALSAMIC#1276]
**Description**

I don't know if this is an issue or not. But clinicians tend to filter our variant-lists based on genes of interest, I think this generally is done in Scout, however maybe some customers are also doing it on their own. 

Gene-names are changed however, and many of them have aliases, and the problem of course is that as long as this is not taken into account, filters using gene-lists may miss some variant which has been annotated with a different gene-name.

I don't know how we're currently avoiding the risks of something like that happening.

How is this done in Scout? People can create gene-lists in Scout. But I'm guessing that this is mostly like a list of strings that are then used to filter against the gene-ids that have been annotated in the uploaded VCF. 

If this is a real issue then I think it could be nice to have some sort of interface where clinicians could paste in genes of interest, and then check this against the list of genes used to annotate variants in any given pipeline. Or maybe we could provide a list of genes used in annotation to Caesar and leave it up to them to verify that their genes are there (but this sounds annoying).

**Suggested solution**

_Write here if there is a suggested way to solve the issue_

**This can be closed when:**

_Describe what needs to be done for this issue to be closed_

**Blocked by**

_If there are any blocking issues/prs/things in this or other repos. Please link to them._

**Before submitting**
- [ ] Assign Effort/Gain/Urgency labels

**Comment by mathiasbio** [comment]
Wrote to Scout developers and if I understood it correctly they have a quite sophisticated system where they start by looking for the HGNC IDs, and if they can't find this ID in the VCF they also check for the gene-ids and associated aliases. So that should reduce the risk of the in silico gene-panels missing anything due to differences in scout / balsamic for gene-names as our VEP annotation also provides the HGNC_IDs.

**Comment by mathiasbio** [comment]
I don't know if I'm ready to close this issue just yet. I'll leave it open to look into further in the future.
